src/resources/assets/js/mixins/largoContainer.vue

```diff
@@ -35,8 +35,20 @@ export default {
             lastSelectedImage: null,
             forceChange: false,
             waitForSessionId: null,
+            showAnnotationOutlines: true,
         };
     },
+    provide() {
+        const appData = {}
+
+        // Need defineProperty to maintain reactivity.
+        // See https://stackoverflow.com/questions/65718651/how-do-i-make-vue-2-provide-inject-api-reactive
+        Object.defineProperty(appData, "showAnnotationOutlines", {
+            get: () => this.showAnnotationOutlines,
+        })
+
+        return { 'outlines': appData };
```

Comment:
Looks good! I'm curious because I never used prop drilling before. Can't you just return the reactive property of the container instance like this?
```js
return {
   outlines: this.showAnnotationOutlines,
};

Reply:
I tried that, but then outlines is not reactive any more.

Reply:
Ok :+1: